rpm.mk

```diff
@@ -47,32 +47,35 @@ bundle-rust:
 
 install-node-modules:
 ifeq ($(COCKPIT_ON), 1)
-	cd src/cockpit/389-console; make -f node_modules.mk install
+	cd src/cockpit/389-console; \
+	rm -rf node_modules; \
+	rm -rf node_modules.release; \
+	npm ci > /dev/null
 endif
 
 build-cockpit: install-node-modules
 ifeq ($(COCKPIT_ON), 1)
+ifndef SKIP_AUDIT_CI
+	cd src/cockpit/389-console; \
+	npm audit
```

Comment:
If I understand correctly, `npm audit` will just check, but it won't stop the execution. We need `npm audit-ci` for complete control here. More info: https://www.npmjs.com/package/audit-ci
So if we have any critical vulnerability issues, our CI will fail immediately.

Reply:
There is no "npm audit-ci" anymore, but there is a "npx- audit-ci".  It seems to work correctly.